modules/coreos-layering-removing.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * post-installation_configuration/coreos-layering.adoc
+
+:_content-type: PROCEDURE
+[id="coreos-layering-removing_{context}"]
+= Removing a {op-system} custom layered image
+
+You can easily revert {op-system-first} image layering from the nodes in specific machine config pools. The Machine Config Operator (MCO) reboots those nodes with the cluster base {op-system-first} image, overriding the custom layered image.
+
+To remove a {op-system-first} custom layered image from your cluster, you need to delete the machine config that applied the image.
+
+.Procedure
+
+. Delete the machine config file that applied the custom layered image.
```

Comment:
Maybe we could say

"Delete the machine config that applied the custom layered image."

Instead of "Delete the machine config file that applied the custom layered image."

Because the MachineConfig is not a file.